cluster/log-dump/log-dump.sh

```diff
@@ -282,7 +282,7 @@ function dump_nodes_with_logexporter() {
   sed -i'' -e "s@{{.EnableHollowNodeLogs}}@${enable_hollow_node_logs}@g" "${KUBE_ROOT}/cluster/log-dump/logexporter-daemonset.yaml"
 
   # Create the logexporter namespace, service-account secret and the logexporter daemonset within that namespace.
-  KUBECTL="${KUBECTL:-${KUBE_ROOT}/cluster/kubectl.sh}"
+  KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
```

Comment:
When it is set to "./cluster/kubectl.sh --match-server-version" (like here - https://github.com/kubernetes/test-infra/blob/master/kubetest/e2e.go#L33), logexporter would fail if there actually is a skew. IIUC we don't want (for the purpose of logexporter) to fail because of the skew, as we just care about uploading the logs (even if there's a skew).